analysis/cross_partition_combiners.py

```diff
@@ -282,33 +278,59 @@ class CrossPartitionCombiner(pipeline_dp.combiners.Combiner):
     # 1. The sum of non dp metrics, which is used for averaging of error
     # metrics.
     # 2. metrics.UtilityReport contains error metrics.
-    AccumulatorType = Tuple[Tuple, metrics.UtilityReport]
-
-    def __init__(self, dp_metrics: List[pipeline_dp.Metrics],
-                 public_partitions: bool):
+    AccumulatorType = Tuple[Tuple, metrics.UtilityReport, Tuple[float]]
```

Comment:
Please extend the comment above about `AccumulatorType` structure

Reply:
added comment, but better to improve it. will do.